### PR TITLE
Updated the tsv files with the updated text

### PR DESCRIPTION
**Pull request by harry6396**
|PR checklist|Status|
|-----|:------:|
|Is issue created in GitHub?|[Yes](https://github.com/maqsoftware/Pehla-School-Urdu-Assets/issues/15)|
|Is code analysis task executed and issues resolved in the Java/Kotlin code?|NA|
|Is code analysis task executed and issues resolved in the CPP/native code?|NA|
|Is the version code set in the AndroidManifest.xml or build.gradle file to 1 more than the version which has already been published?|NA|
|Is code commenting done properly for the changes made?|NA|
|Is correct variable naming convention followed?|NA|
|Is auto formatting shortcut Ctrl+Alt+Shift+L executed in all files?|NA|
|Are the message strings used in the application taken from strings.xml file?|NA|
|Are the permissions which are not required removed from AndroidManifest.java file?|NA|
|Is the _uses-feature_ tag used for only must have feature in AndroidManifest.java file?|NA|
|Are the generated files correctly configured in .gitignore?|NA|
|Are the release signing configurations removed from the gradle files?|NA|
|Is commented code removed from all files?|NA|
|Are updated images compressed (< 100 KB)?|NA|
|Ensured there are no typos in any string. For example: Variable/method names, hardcoded strings, commit messages, PR titles, issues, code comments?|NA|

OBB

|PR checklist|Status|
|-----|:------:|
|Is size of the OBB correct?|NA|
|Is OBB file version correctly set in DownloadExpansionFile.java file?|NA|
|Is patch OBB file to be deployed? If yes, then are the settings correct in DownloadExpansionFile.java file?|NA|

### CODE REVIEW

File: localized/en-us/games/eggquiz/eggquizmath_levels.tsv [hunk 1]
```diff
@@ -49,34 +49,34 @@ PostTest_3	1	3001, 3002, 3003~3004, 3005, 3006, 3007~3008, 3009, 3010, 3011, 301
...
 			42001	2digit_numbers	कितने हैं?					51~100				
-			42002	compare_number_magnitudes	ascending	سب سے چھوٹی سے نمبروں کو آرڈر.	सबसे छोटे से लेकर सबसे बड़े तक के क्रम में संख्या को रखे			1~20				
+			42002	compare_number_magnitudes	ascending	اعداد کو چھوٹے سے بڑے کی ترتیب دیں	सबसे छोटे से लेकर सबसे बड़े तक के क्रम में संख्या को रखे			1~20				
 			42003	operations_with_objects	2~10-1~4=?					1~9				
 			42004	operations_with_objects	2~10-5~9=?					1~9				
-			42005	shapes	which	کون سا دائرہ ہے ؟	कौनसा सरकल है?			circle	circle,square,triangle,rectangle			
-			42006	shapes	which	کون سا مربع ہے ؟	कौनसा स्क्वायर है?			square	circle,square,triangle,rectangle			
-			42007	shapes	which	کون مثلث ہے ؟	कौनसा ट्रायंगल है?			triangle	circle,square,triangle,rectangle			
-			42008	shapes	which	کون سا مستطیل ہے ؟	कौन सा रैक्टेंगल है?			rectangle	circle,square,triangle,rectangle			
+			42005	shapes	which	کون سی شکل "سرکل" ہے؟	कौनसा सर्कल है?			circle	circle,square,triangle,rectangle
```
Comment (akashdabhi03) [inline localized/en-us/games/eggquiz/eggquizmath_levels.tsv]: Revert back to `सरकल`. `सर्कल` is not pronounced correctly by TTS.

File: localized/en-us/games/eggquiz/eggquizmath_levels.tsv [hunk 2]
```diff
@@ -49,34 +49,34 @@ PostTest_3	1	3001, 3002, 3003~3004, 3005, 3006, 3007~3008, 3009, 3010, 3011, 301
...
 			4007	missing_number_drag	1	4	descending			1~20				
 			4008	missing_number_drag	2	3	ascending			1~20	2			
 			4009	operations_with_objects	1~20+1~20=?	regroupingfalse				1~20				
 			4010	operations_with_objects	1~20-1~20=?	regroupingfalse				1~20				
-			4011	shapes	which	کون سا دائرہ ہے ؟	कौनसा सरकल है?			circle	circle,square,triangle,rectangle			
-			4012	shapes	which	کون سا مربع ہے ؟	कौनसा स्क्वायर है?			square	circle,square,triangle,rectangle			
-			4013	shapes	which	کون مثلث ہے ؟	कौनसा ट्रायंगल है?			triangle	circle,square,triangle,rectangle			
-			4014	shapes	which	کون سا مستطیل ہے ؟	कौन सा रैक्टेंगल है?			rectangle	circle,square,triangle,rectangle			
-			4015	shapes	what	یہ شکل کیا ہے ؟	यह कौन सा आकार है?			circle,square,triangle,rectangle	circle,square,triangle,rectangle			
+			4011	shapes	which	کون سی شکل "سرکل" ہے؟	कौनसा सर्कल है?			circle	circle,square,triangle,rectangle
```
Comment (akashdabhi03) [inline localized/en-us/games/eggquiz/eggquizmath_levels.tsv]: Revert to `सरकल`. Replace in the whole file.

File: localized/en-us/games/wordwindow/wordwindow_level.tsv [hunk 3]
```diff
@@ -187,55 +187,55 @@ en-US	14	1	5	4	"बच्चे अपनी कक्षा को सजान
...
+en-US	17	3	2	5	"सारा ने 6 फूल चुने। उसने फूलो की समान मात्रा मे 2 फूलदानो मे डाल दी। प्रत्येक फूलदान मे उसने कितने फूल डाले ?"	سارہ نے 6 پھول اٹھایا۔ اسنے دو گلدان میں پھولوں کی برابر تعداد ڈال دی۔ کتنے پھول اس نے ہر گلدان میں ڈالی؟	2	3	5	6	flowers	flowers	flowers	flowers	6 = <3> + <3>	2	3	3	0	0					wordwindow-stone-yellow.png	wordwindow-stone-blue.png		
+en-US	17	3	3	5	"एक किसान के पास 16 अंडे है और वो उन्हे समान रूप से 2 बक्से मे रखता है। प्रत्येक बॉक्स मे कितने अंडे है?"	ایک کسان کے پاس 16 انڈے ہیں اور وہ ان کو دو بکس میں یکساں طور پر رکھتا ہے۔ کتنے انڈے ہر بکس میں ہیں؟	2	4	8	16	eggs	eggs	eggs	eggs	16 = <8> + <8>	3	8	8	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
+en-US	17	3	4	5	"शेन ने एक पेड़ से 6 आम निकाले। उन्होने आमो की एक समान संख्या को 3 बक्से मे रखा। प्रत्येक डि4बे मे उसने कितने आम रखे?"	شانٹی ایک درخت سے 6 آم توڑے۔ اسنے تین بکس میں آم کی ایک برابر تعداد ڈال دی۔ اسنے کتنے آم ہر بکس میں ڈالے؟	2	3	5	6	mangoes	mangoes	mangoes	mangoes	6 = <2> + <2> + <2>	1	2	2	2	0					wordwindow-stone-red.png	wordwindow-stone-green.png	wordwindow-stone-blue.png
```
Comment (akashdabhi03) [inline localized/en-us/games/wordwindow/wordwindow_level.tsv]: Fix डि4बे

File: localized/en-us/games/wordwindow/wordwindow_level.tsv [hunk 4]
```diff
@@ -187,55 +187,55 @@ en-US	14	1	5	4	"बच्चे अपनी कक्षा को सजान
...
+en-US	17	3	3	5	"एक किसान के पास 16 अंडे है और वो उन्हे समान रूप से 2 बक्से मे रखता है। प्रत्येक बॉक्स मे कितने अंडे है?"	ایک کسان کے پاس 16 انڈے ہیں اور وہ ان کو دو بکس میں یکساں طور پر رکھتا ہے۔ کتنے انڈے ہر بکس میں ہیں؟	2	4	8	16	eggs	eggs	eggs	eggs	16 = <8> + <8>	3	8	8	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
+en-US	17	3	4	5	"शेन ने एक पेड़ से 6 आम निकाले। उन्होने आमो की एक समान संख्या को 3 बक्से मे रखा। प्रत्येक डि4बे मे उसने कितने आम रखे?"	شانٹی ایک درخت سے 6 آم توڑے۔ اسنے تین بکس میں آم کی ایک برابر تعداد ڈال دی۔ اسنے کتنے آم ہر بکس میں ڈالے؟	2	3	5	6	mangoes	mangoes	mangoes	mangoes	6 = <2> + <2> + <2>	1	2	2	2	0					wordwindow-stone-red.png	wordwindow-stone-green.png	wordwindow-stone-blue.png		
+en-US	17	3	5	5	"20 छात्रो को शिक्षक द्वारा 4 समूहो मे समान रूप से विभाजित किया गया था। प्रत्येक समूह मे कितने छात्र है?"	20طالب علموں کو استاد کی طرف سے 4 گروپوں میں تقسیم کیا گیا۔ کتنے طلباء ہرایک گروپ میں ہیں؟	5	10	15	20	students	students	students	students	20 = <5> + <5> + <5> + <5>	1	5	5	5	5					wordwindow-stone-blue.png	wordwindow-stone-red.png	wordwindow-stone-green.png	wordwindow-stone-yellow.png
```
Comment (akashdabhi03) [inline localized/en-us/games/wordwindow/wordwindow_level.tsv]: Is `2020` correct? Please check.

File: localized/en-us/games/wordwindow/wordwindow_level.tsv [hunk 5]
```diff
@@ -15,109 +15,109 @@ en-US	1	3	3	1	"जब वह स्कूल में गणित करती
...
+en-US	3	3	5	2	"एना के पास 7 बिस्कुट है। वह उनमे से 2 खाती है। उसके पास कितनी बिस्कुट़ बची है?"	انا كے پاس 7 بسکٹ ہیں . وہ اس میں سے 2 کھا لیتی ہے. اب اسکے پاس کتنے بسکٹ بچے ہیں؟	3	4	5	6	biscuits	biscuits	biscuits	biscuits	7 - 2 = <5>	3	7	2	0	0					wordwindow-stone-red.png	wordwindow-stone-blue.png		
+en-US	4	1	1	4	"6 पक्षी और 3 घोसले है। घोसलो की तुलना मे कितने अधिक पक्षी है"	6پرندے اور 3 گھوسلے ہیں. گھوسلوں کی تُلْنا  میں کتنے پرندے زیادہ ہیں؟	3	6	9	12	birds	birds	birds	birds	6 - 3 = <3>	1	6	3	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
+en-US	4	1	2	4	"एमी ने 4 ब्लॉक चलती है स्कूल के लिए। जेन स्कूल के लिए 2 4लॉक चलता है। एमी जेन की तुलना मे स्कूल जाने के लिए और कितने 4लॉक चलती है?"	ایمی اسکول جانے كے لیے 4 بلا ک چلتی ہے. جین اسکول جانے كے لیے 2 بلا ک چلتی ہے. ایمی جین کی تُلْنا میں کتنا بلا ک زیادہ چلتی ہے؟	2	4	6	8	blocks	blocks	blocks	blocks	4 - 2 = <2>	1	4	2	0	0					wordwindow-stone-blue.png	wordwindow-stone-red.png
```
Comment (akashdabhi03) [inline localized/en-us/games/wordwindow/wordwindow_level.tsv]: 4लॉक -> block

File: localized/en-us/games/wordwindow/wordwindow_level.tsv [hunk 6]
```diff
@@ -15,109 +15,109 @@ en-US	1	3	3	1	"जब वह स्कूल में गणित करती
...
+en-US	8	1	4	5	"हार्पर और रेवेन ने एक साथ 8 तस्वीरे खीची। हार्पर ने 3 तस्वीरे खीचे। रेवेन ने कितने तस्वीरे खीचे?"	ہارپراور ریوین نے 8 تصویریں ایک ساتھ بنائی ہیں. ہارپر نے 3 تصویریں بنائ ہیں۔ ریوین نے کتنی تصویریں بنائ ہیں؟	3	5	8	11	pictures	pictures	pictures	pictures	8 = 3 + <5>	2	3	5	0	0					wordwindow-stone-blue.png	wordwindow-stone-yellow.png		
+en-US	8	1	5	5	"केटी और उसके भाई ने 6 आमो को उठाया। केटी ने 4 आम उठाए। उसके भाई ने कितने आम उठाए?"	کیٹی اور اس کے بھائی نے ملکر 6 آم اٹھاے. کیٹی نے 4 آم اٹھائ. اس کے بھائی نے کتنے آم اٹھاے؟	2	4	6	8	mangoes	mangoes	mangoes	mangoes	6 = 4 + <2>	1	4	2	0	0					wordwindow-stone-red.png	wordwindow-stone-green.png		
+en-US	8	2	1	5	"जेन के पास 9 गुब्बारे है। 6 ग्रीन  है और बाकी 4लू है। 4लू  गुब्बारे कितने है?"	جین کے پاس 9 غببارے ہیں. 6 سبز ہیں اور باقی نیلے ہیں. کتنے غببارے نیلے ہیں؟	2	3	4	5	balloons	balloons	balloons	balloons	9 = 6 + <3>	2	6	3	0	0					wordwindow-stone-green.png	wordwindow-stone-blue.png
```
Comment (akashdabhi03) [inline localized/en-us/games/wordwindow/wordwindow_level.tsv]: 4लू -> blue

File: localized/en-us/games/wordwindow/wordwindow_level.tsv [hunk 7]
```diff
@@ -187,55 +187,55 @@ en-US	14	1	5	4	"बच्चे अपनी कक्षा को सजान
...
+en-US	15	1	5	5	"सुश्री डॉसन की कक्षा मे 12 छात्र है। कुछ छात्र अनुपस्थित है और 7 छात्र उपस्थित है। कितने छात्र अनुपस्थित है?"	محترمہ ڈاونسن کی کلاس میں کل 12 طالب علم ہیں۔ 7 طالب علم حاضر ہیں اور کچھ غیر حاضر ہیں۔ کتنے طلباء غیر حاضر ہیں؟	5	7	9	12	students	students	students	students	12 = <5> + 7	1	5	7	0	0					wordwindow-stone-yellow.png	wordwindow-stone-green.png		
+en-US	15	2	1	5	"टेनिस टीम मे 12 खिलाड़ी है। खिलाड़यिो मे से 6 लड़किया है। कुछ लडक़े है। कितने टेनिस खिलाड़ी लडक़े है?"	ٹینس ٹیم میں 12 کھلاڑی ہیں۔ جنمیں 6 لڑکیاں ہیں اور کچھ لڑکے ہیں۔ ٹیم میں کتنے کھلاڑی لڑکے ہیں؟	2	6	10	12	boys	boys	boys	boys	12 = 6 + <6>	2	6	6	0	0					wordwindow-stone-blue.png	wordwindow-stone-red.png		
+en-US	15	2	2	5	"लकड़ी का एक टुकड़ा 13 इच लबा होता है। इसे दो भागो मे काटा जाता है। एक भाग 6 इच लबा है। दूसरा भाग कितना ल6बा है?"	لکڑی کا ایک ٹکڑا 13 انچ لمبا ہے۔ اسکو 2 حصّوں میں کاٹ دیا گیا۔ ایک حصہ 6 انچ لمبا ہے۔ دوسرا حصہ کتنا لمبا ہے؟	6	7	8	9	inches	inches	inches	inches	13 = 6 + <7>	2	6	7	0	0					wordwindow-stone-green.png	wordwindow-stone-red.png
```
Comment (akashdabhi03) [inline localized/en-us/games/wordwindow/wordwindow_level.tsv]: ल6बा -> lamba

File: localized/en-us/games/eggquiz/eggquizmath_levels.tsv (hunk 1 above)
Comment (harry6396) [reply localized/en-us/games/eggquiz/eggquizmath_levels.tsv]: Reverted

**Pull request merged**